2주차/23827/23827_python_leo.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+answer = 0
+N = int(sys.stdin.readline())
+array = list(map(int, sys.stdin.readline().split()))
+total = sum(array)
+for i in array[:-1]:
+    total -= i
+    answer += i * total
```

Comment:
배열의 마지막부분부터 연산하지 않아도 통과하는데 뒤에서 부터 계산한 이유가 있을까요?
[:-1] 부분은 빼도 될 것 같아요

```suggestion
for i in array:
    total -= i
    answer += i * total
```

Reply:
array[:-1]는 배열의 마지막요소를 반복문에서 제외하려는 의도였고 배열의 순서와는 관련없는 걸로 알고 있습니다.
문제를 풀때는 마지막 요소는 계산할 필요가 없다고 생각해서 반복문에서 뺐던건데 total이 0이여서 0 * x가 되니 제외하지 않아도 되는군요.

Reply:
아아 [::-1] 와 혼동했네요 고마워요 레오!